🐛 Import package issue on `v0.2.2`

It seems since the version `0.2.2` (published a few hours ago from this post) the package is not recognized anymore in JS/TS projects.

I haven't dug into it yet, but I've just tried importing it into 2 different repo and I got the same issue:

- code: `import { DuneClient } from "@duneanalytics/client-sdk";`
⬇️
- error: `Cannot find module '@duneanalytics/client-sdk' or its corresponding type declarations.`

Before `v0.2.2` I didn't have any issue with the package.

--- a/package.json
+++ b/package.json
@@ -4,8 +4,9 @@
   "author": "Ben Smith <bh2smith@gmail.com>",
   "description": "Node Client for Dune Analytics' officially supported API.",
   "repository": "git@github.com:duneanalytics/ts-dune-client.git",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
   "files": [
     "dist/**/*"
   ],

